ql/src/java/org/apache/hadoop/hive/ql/exec/Utilities.java

```diff
@@ -800,6 +800,7 @@ public static TableDesc getTableDesc(String cols, String colTypes) {
         serdeConstants.SERIALIZATION_FORMAT, "" + Utilities.ctrlaCode,
         serdeConstants.LIST_COLUMNS, cols,
         serdeConstants.LIST_COLUMN_TYPES, colTypes,
+            hive_metastoreConstants.TABLE_BUCKETING_VERSION, "-1",
```

Comment:
nit. indentation

Reply:
removed the usage of this makeProperties